psi4/driver/procrouting/proc.py

```diff
@@ -1636,12 +1646,15 @@ def scf_helper(name, post_scf=True, **kwargs):
                  scf_wfn.epsilon_b(), scf_wfn.occupation_a(),
                  scf_wfn.occupation_b(), dovirt)
 
-    # Write out orbitals and basis; Can be disabled, e.g., for findif displacements
-    if kwargs.get('write_orbitals', True):
-        write_filename = scf_wfn.get_scratch_filename(180)
-
-        scf_wfn.to_file(write_filename)
-        extras.register_numpy_file(write_filename)
+    # Write checkpoint file (orbitals and basis); Can be disabled, e.g., for findif displacements
+    if write_checkpoint_file and isinstance(_chkfile, str):
+        filename = kwargs.get('write_orbitals')
```

Comment:
```suggestion
        filename = kwargs['write_orbitals']
```

We want a crash if this somehow isn't set.